packages/fhir-router/src/fhirrouter.ts

```diff
@@ -26,7 +32,16 @@ export type FhirRequest = {
 
 export type FhirResponse = [OperationOutcome] | [OperationOutcome, Resource];
 
-export type FhirRouteHandler = (req: FhirRequest, repo: FhirRepository, router: FhirRouter) => Promise<FhirResponse>;
+export type FhirRouteOptions = {
+  batch?: boolean;
+};
+
+export type FhirRouteHandler = (
```

Comment:
I was thinking about this more, and I think we should have a team session on this, because it's going to be a pretty central concept throughout the entire code base.

This doesn't necessarily need to be part of this PR, just thinking out loud.

A couple thoughts:
1. Given that we're already at 4 args, I suspect we should just make this into a single argument "context" type
2. We may want to consider adding (moving?) a bunch of the `AuthenticatedRequestContext` properties to that "context" type
3. As you've pointed out in the past, `FhirResponse` should probably be an object rather than an array.